.travis.yml

```diff
@@ -1,5 +1,4 @@
 sudo: false
-cache: bundler
```

Comment:
I prefer to not adde other development dependency in the project. If travis can handle it, I don't think we need a dependency to do it again.